recipes-graphics/imx-gpu-g2d/imx-gpu-g2d_6.2.4.p1.6.bb

```diff
@@ -35,4 +35,5 @@ FILES_${PN} = "${libdir}/libg2d* /opt"
 FILES_${PN}-dev = "${includedir}"
 INSANE_SKIP_${PN} = "ldflags"
 
-COMPATIBLE_MACHINE = "(mx6|mx7ulp)"
+COMPATIBLE_MACHINE = "(^$)"
+COMPATIBLE_MACHINE_imxgpu2d = "(mx6|mx7)"
```

Comment:
Why change it? it adds an indirection and it is harder to read.

Reply:
I see. In this case, we have two options:

1. list all compatible SoCs
2. use the imxgpu2d override feature. 

If we take the second, which is more or less what you did, it can be simplified as:

```
COMPATIBLE_MACHINE = "(imxgpu2d)"
```

Reply:
Right, good idea.